src/themes/website/styles/widgets/widgets.scss

```diff
@@ -18,3 +18,31 @@
 @import "fui/operation-details.scss";
 @import "fui/apis-products-lists.scss";
 @import "fui/reports.scss";
+
+/**
+ * If you need to make sure that custom defined styles do not interfere with widgets,
+ * this is a good place to do it.
+ * Note: selector is duplicated to increase its weight. Do not use !important as it will break
+ * some widgets that rely on inline styles.
+ */
+.fui-FluentProvider.fui-FluentProvider.fui-FluentProvider {
+  .button {
+    margin: 10px 20px 10px 0;
+    padding: 9px 50px;
```

Comment:
I am still sure this 50px value is wrong for the redesign. I know you took it from old widgets. Please, check designs and come up with proper spacing. For everything you specify here, not just this padding.